recipes/recipe-lemongrass-portobello-steak.html

```diff
@@ -0,0 +1,170 @@
+                <div class="recipe_body right">
+                    <div>
+                        <h3>Recipe submitted by Ann</h3>
+                        <p class="recipe_info">
+                            <i class="fas fa-seedling"></i>Vegan&ensp;   
+    <!--                        <i class="fas fa-carrot"></i>Vegetarian&ensp;-->
+                            <i class="fas fa-ban"></i>Gluten-Free&ensp;
+                            <i class="fas fa-circle"></i>Easy&ensp;
+    <!--                        <i class="fas fa-square"></i>Moderate&ensp;-->
+    <!--                        <i class="fas fa-star"></i>Difficult&ensp;-->
+                            <br>
+                            <i class="fas fa-clock"></i>30 mins&ensp;
+                            <i class="fas fa-user"></i>3-4 Servings
+                        </p>
+<!--
+                        <p>Description: 
+                            <a href="https://www.youtube.com/watch?v=tUi4_ektVn0" target="_blank"> Video 1</a>
+                        </p>
+-->
+                    </div>
+                </div>
+            </div>
+            
+            
+            <div class="recipe_image_text_section" id="recipe_image_text_section_body">
+                
+                <div class="recipe_text left">
+                    <h3>Ingredients</h3>
+                    <ul>
+                        <li>4 large portobello mushrooms</li>
+                        <li>1-2 Stalk lemongrass</li>
+                        <li>2-3 Kaffir lime leaves</li>
+                        <li>2-3 Cilantro stems</li>
+                        <li>1 tbsp coconut amino sauce</li>
+                        <li>1 tsp salt</li>
+                        <li>1/2 tsp chilli powder</li>
+                        <li>1 tsp black pepper</li>
+                        <li>1-2 medium sweet potatoes</li>
+                        <li>1 broccoli head</li>
+                        <li>1 tbsp nutritional yeast (optional) </li>
+            
+                    </ul>
+                </div>
+
+                <div class="recipe_text right">
+                    <h3>Directions</h3>
+                    <ol>
+                        <li>Cut the mushroom into small slices and put them in a big ziplock bag</li>
+                        <li>Finely chop all the fresh spices and pour them into the ziplock bag</li>
+                        <li>Marinate the mushrooms with 1/2 black pepper, 1/2 salt, chilli powder and coconut amino for 5 minutes</li>
+                        <li>Grill the mushroom in a pan until it's cooked and set aside</li>
+                        <li>Steam the broccoli and sweet potato for 5-10 minutes or until soft</li>
+                        <li>Mash the broccoli and sweet potato in the food processor or use a fork to mash them</li>
+                        <li>Add 1/2 salt, 1/2 black pepper and the nutritional yeast to the mashed broccoli and sweet potato</li>
+                        
```

Comment:
```suggestion
                        <li>Add 1/2 salt, 1/2 black pepper and the nutritional yeast to the mashed broccoli and sweet potato</li>
                        <li>Serve with romaine lettuce and any topping you like. I used cilantro, tomato and diced pineapple, it taste awesome! /li>                        
```

Reply:
The "Additional Information" is a direction this time, so it can go in this section